app/components/ListClaimConditions/ListClaimConditions.tsx

```
@@ -0,0 +1,339 @@
+                    variant="ghost"
+                    disabled={true} // TODO: Need to visit when update claim condition function is resolved
+                    leftIcon={
+                      editStates[i] ? (
+                        <CloseIcon boxSize={3} />
+                      ) : (
+                        <EditIcon boxSize={3} />
+                      )
+                    }
+                    onClick={() => {
+                      return; // FIXME: Remove when update claim condition function is resolved
+                      toggleEditClaim(i);
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)